tsl/src/chunk.c

```diff
@@ -259,3 +260,94 @@ chunk_invoke_drop_chunks(Oid relid, Datum older_than, Datum older_than_type)
 
 	return num_results;
 }
+
+/*
+ * drop_chunk_replica:
+ *
+ * This function drops a chunk on a specified data node. It then
+ * removes the metadata about the datanode, chunk association on the
```

Comment:
```suggestion
 * removes the metadata about the data node, chunk association on the
```

I am confused by this comment. Will it be correct to say?:
```suggestion
 * removes the metadata about the association of the chunk to the data node on the
```

Reply:
Done